python_webapp2_vanilla/main.py

```diff
@@ -0,0 +1,16 @@
+import webapp2
+
```

Comment:
No it doesn't need webapp2 since it is already installed in the custom base Image that I have built for this (jin09/webapp2)